README.md

```diff
@@ -212,27 +201,28 @@ specific label files (see section below). Please type:
 
 to see all the options.
 
-Note that to replicate results exactly as in the paper you may need to checkout
-the specific version of NuPIC (and associated nupic.core) that is noted in the
-[Scoreboard](https://github.com/numenta/NAB/wiki/NAB%20Scoreboard):
+##### Run HTM with NAB
+
+In 2019 much of NAB was ported to Python 3, however NuPIC itself was not.
+
+NuPIC dependent detectors are available in the [NAB-detectors]() repository.
```

Comment:
TODO please add link

Reply:
I am going to add some more detail to this before we merge. I think it should be called `htm-nab-detectors`.